Updated installer files from master to main.

marcpopMSFT opened the pull request:
Added a nuget.config so that the table generator will build.

sfoslund commented:
@marcpopMSFT is this good to merge?

akoeplinger commented on tools/sdk-readme-table-generator/TableGenerator/Program.fs:
```diff
@@ -6,8 +6,8 @@ open TableGenerator.Reference
 let inputBranches =
-        [ { GitBranchName = "master"
-            DisplayName = "Master<br>(6.0.x&nbsp;Runtime)"
+        [ { GitBranchName = "Main"
```
this should be lowercase `main`, otherwise the generated link doesn't work.

akoeplinger commented on tools/sdk-readme-table-generator/TableGenerator/Program.fs:
```diff
@@ -32,12 +32,12 @@ let inputBranches =
 > **1**: Our Debian packages are put together slightly differently than the other OS specific installers. Instead of combining everything, we have separate component packages that depend on each other. If you're installing the SDK from the .deb file (via dpkg or similar), then you'll need to install the corresponding dependencies first:
 > * [Host, Host FX Resolver, and Shared Framework](https://github.com/dotnet/runtime#daily-builds)
-> * [ASP.NET Core Shared Framework](https://github.com/aspnet/AspNetCore/blob/master/docs/DailyBuilds.md)
+> * [ASP.NET Core Shared Framework](https://github.com/aspnet/AspNetCore/blob/Main/docs/DailyBuilds.md)
```
same here

akoeplinger commented on tools/sdk-readme-table-generator/TableGenerator/Program.fs:
```diff
@@ -32,12 +32,12 @@ let inputBranches =
 
 let sdksha2 =
-    "[sdk-shas-2.2.1XX]: https://github.com/dotnet/versions/tree/master/build-info/dotnet/product/cli/release/2.2#built-repositories"
+    "[sdk-shas-2.2.1XX]: https://github.com/dotnet/versions/tree/Main/build-info/dotnet/product/cli/release/2.2#built-repositories"
```
and here

akoeplinger commented on tools/sdk-readme-table-generator/TableGenerator/Shared.fs:
```diff
@@ -22,14 +22,14 @@ type BranchMajorMinorVersion =
-    match branch.GitBranchName = "master" with
-    | true -> Master
+let getMajorMinor (branch: Branch): BranchMajorMinorVersionOrMain =
+    match branch.GitBranchName = "Main" with
```
and here

akoeplinger commented on tools/sdk-readme-table-generator/TableGenerator.Tests/DomainTests.fs:
```diff
@@ -10,12 +10,12 @@ open Xunit
     let branch =
-        { GitBranchName = "master"
-          DisplayName = "Master<br>(5.0.x&nbsp;Runtime)"
+        { GitBranchName = "Main"
```
and here

akoeplinger commented on tools/sdk-readme-table-generator/TableGenerator.Tests/DomainTests.fs:
```diff
@@ -10,12 +10,12 @@ open Xunit
+          DisplayName = "Main<br>(5.0.x&nbsp;Runtime)"
           AkaMsChannel = Some("net5/dev") }
-    branchNameShorten branch |> should equal "master"
+    branchNameShorten branch |> should equal "Main"
```
and here. will stop commenting now, but these need to be fixed everywhere :)

akoeplinger reviewed (changes requested):
There's a casing issue in the renamed branch that needs to be fixed.

marcpopMSFT commented:
@akoeplinger thanks for catching that. I'd switched to upper case in the table file to not conflict with the main method in program.fs but I was too broad with that fix. I've updated now and I think I fixed all the broken spots while leaving the table generator still passing all tests.
